src/class_singleCleaner.py

```diff
@@ -48,20 +48,15 @@ class singleCleaner(StoppableThread):
     def run(self):  # pylint: disable=too-many-branches
         gc.disable()
         timeWeLastClearedInventoryAndPubkeysTables = 0
-        try:
-            state.maximumLengthOfTimeToBotherResendingMessages = (
-                float(BMConfigParser().get(
-                    'bitmessagesettings', 'stopresendingafterxdays'))
-                * 24 * 60 * 60
-            ) + (
-                float(BMConfigParser().get(
-                    'bitmessagesettings', 'stopresendingafterxmonths'))
-                * (60 * 60 * 24 * 365) / 12)
-        except:
-            # Either the user hasn't set stopresendingafterxdays and
-            # stopresendingafterxmonths yet or the options are missing
-            # from the config file.
-            state.maximumLengthOfTimeToBotherResendingMessages = float('inf')
+
+        state.maximumLengthOfTimeToBotherResendingMessages = (
+            float(BMConfigParser().safeGetFloat(
```

Comment:
Why do you cast float to float?

Reply:
tested, additional cast not needed, please remove

Reply:
Updated